tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -128,11 +129,11 @@ def _create_find_statistics_snapshot(self) -> str:
                 except NoSuchElementException:
                     publication_summary = ""
 
-                release_type = publication.find_element(By.XPATH, "./dl//dd[@data-testid='release-type']").text
+                release_type = publication.find_element(By.XPATH, "./dl//dd[@data-testid='Release type-value']").text
```

Comment:
Feel free to ignore if it's not important, but the other `testid` formats shown here are snake-cased, i.e. `Release-type-value` instead of `Release type-value` (don't know if this matters).